snuba/settings/__init__.py

```diff
@@ -245,6 +245,7 @@
 
 # File path glob for configs
 STORAGE_CONFIG_FILES_GLOB = f"{CONFIG_FILES_PATH}/**/storages/*.yaml"
+ENTITY_CONFIG_FILES_GLOB = f"{CONFIG_FILES_PATH}/**/entities/*.yaml"
```

Comment:
We're going to need a better way of doing this I think. I feel like this works now because there is only one folder in the config file path.